### PR TITLE
[Add] User model

### LINKED ISSUE #1: Create User Model
Create the model with all attributes in `/backend/models.py`

- `name`* (string)
- `email`* (email)
- `password`* (string)
- `phone` (number)
- `addresses` ( [string] )

Note:
- `*` means required
- All arrays are empty by default
- All non-required numbers are 0 by default

### PR DESCRIPTION
**Pull request by gsc2001**
Fixes #1

### CODE REVIEW

File: backend/ecommerce/models.py [hunk 1]
```diff
@@ -1,7 +1,36 @@
...
+    is_staff = models.BooleanField(
+        default=False, help_text=_("Designates if a user can log into this admin site")
+    )
+    phone = models.IntegerField(
+        validators=[MinValueValidator(1000000000), MaxValueValidator(9999999999)]
+    )
+
+    USERNAME_FIELD = "email"
+
+    REQUIRED_FIELDS = ["name", "phone"]
```
Comment (tichnas) [inline backend/ecommerce/models.py:21]: Isn't everything required by default?

**Comment by tichnas** [comment]
Should we add `backend/ecommerce/migrations/` in gitignore ?

File: backend/ecommerce/models.py (hunk 1 above)
Comment (gsc2001) [reply backend/ecommerce/models.py:21]: These are the fields which will be asked while creating the super user.

**Comment by gsc2001** [comment]
No you need to commit (https://docs.djangoproject.com/en/3.0/topics/migrations/#version-control)

**Review by tichnas — APPROVED** [review]

**Pull request merged**